tests/unit/test_config.py

```diff
@@ -81,3 +81,16 @@ def test_load_unicode_error(tmp_dir, dvc, mocker):
     with pytest.raises(ConfigError):
         with config.edit():
             pass
+
+
+def test_load_configob_error(tmp_dir, dvc, mocker):
+    from configobj import ConfigObjError
+
+    config = Config(validate=False)
+    mocker.patch(
+        "configobj.ConfigObj",
+        side_effect=ConfigObjError(),
+    )
```

Comment:
[Don’t Mock What You Don’t Own](https://hynek.me/articles/what-to-mock-in-5-mins/)

Reply:
Just copied from the already existing test above 🤷

Reply:
yeah, what are they going to do? Make a new release? Probably not. 😄